.github/workflows/sematic-release.yaml

```diff
@@ -17,19 +17,37 @@ jobs:
     steps:
```

Comment:
I think we can remove this.  We couldn't get Sematic-release + mono repo to run on CI, it would error out 

https://github.com/pmowrer/semantic-release-monorepo

Reply:
Remove the entire Github Action @QuinnLee ?

Reply:
yes!